src/main/webapp/loginpage.xhtml

```diff
@@ -194,15 +194,11 @@
                                         </h:outputLink>
                                         <ui:fragment rendered="#{fn:contains(LoginPage.authProvider.id, 'orcid')}">
                                             <p class="help-block small" style="margin-top:1em;">
-                                                ORCID is an open, non-profit, community-based effort to provide a registry of unique researcher identifiers
-                                                and a transparent method of linking research activities and outputs to these identifiers. ORCID is unique in
-                                                its ability to reach across disciplines, research sectors, and national boundaries and its cooperation with
-                                                other identifier systems. Find out more at <a href="https://orcid.org/about" target="_blank">orcid.org/about</a>.
-						</p>
-						<p class="help-block small" style="margin-top:1em;">
-						This repository uses your ORCID for authentication (so you don't need another username/password combination).
-						Having your ORCID associated with your datasets also makes it easier for people to find the datasets you have published.
-						</p>
+                                                <h:outputText value="#{auth.providers.orcid.helpmessage1}"  escape="false"/>
+                                            </p>
+                                            <p class="help-block small" style="margin-top:1em;">
+                                                <h:outputText value="#{auth.providers.orcid.helpmessage2}" />
```

Comment:
@JayanthyChengan Sorry for missing this earlier! This h:outputText component is missing the `escape="false"` attribute, like the previous line of text uses. After that, good to go to QA!

Reply:
@JayanthyChengan You are correct. My mistake, I thought the link was included as part of the second message. That said, to be consistent though, I think it would be wise to include it for both messages. If for whatever reason the text is later shuffled around in the bundle and a link ends up in one or the other or both messages!

Reply:
Sure. committed the latest